app/scripts/modules/core/src/presentation/icons/iconsByName.ts

```diff
@@ -1,9 +1,124 @@
+import { ReactComponent as build } from './vectors/build.svg';
+import { ReactComponent as canary_fail } from './vectors/canary_fail.svg';
+import { ReactComponent as canary_running } from './vectors/canary_running.svg';
+import { ReactComponent as canary_pass } from './vectors/canary_pass.svg';
+import { ReactComponent as canary_marginal } from './vectors/canary_marginal.svg';
+import { ReactComponent as check_badge } from './vectors/check_badge.svg';
+import { ReactComponent as checkbox_indeterminate } from './vectors/checkbox_indeterminate.svg';
+import { ReactComponent as checkbox_checked } from './vectors/checkbox_checked.svg';
+import { ReactComponent as checkbox_unchecked } from './vectors/checkbox_unchecked.svg';
 import { ReactComponent as close } from './vectors/close.svg';
+import { ReactComponent as close_small } from './vectors/close_small.svg';
+import { ReactComponent as cloud_deployed } from './vectors/cloud_deployed.svg';
+import { ReactComponent as cloud_error } from './vectors/cloud_error.svg';
+import { ReactComponent as cloud_progress } from './vectors/cloud_progress.svg';
+import { ReactComponent as cloud_decommissioned } from './vectors/cloud_decommissioned.svg';
+import { ReactComponent as cluster } from './vectors/cluster.svg';
+import { ReactComponent as copy_clipboard } from './vectors/copy_clipboard.svg';
 import { ReactComponent as environment } from './vectors/environment.svg';
+import { ReactComponent as fn } from './vectors/fn.svg';
+import { ReactComponent as fn_new } from './vectors/fn_new.svg';
+import { ReactComponent as instances } from './vectors/instances.svg';
+import { ReactComponent as load_balancer } from './vectors/load_balancer.svg';
+import { ReactComponent as manual_judgement } from './vectors/manual_judgement.svg';
+import { ReactComponent as md_actuating } from './vectors/md_actuating.svg';
+import { ReactComponent as md_actuation_launched } from './vectors/md_actuation_launched.svg';
+import { ReactComponent as md_created } from './vectors/md_created.svg';
+import { ReactComponent as md_delta_detected } from './vectors/md_delta_detected.svg';
+import { ReactComponent as md_delta_resolved } from './vectors/md_delta_resolved.svg';
+import { ReactComponent as md_diff } from './vectors/md_diff.svg';
+import { ReactComponent as md_error } from './vectors/md_error.svg';
+import { ReactComponent as md_flapping } from './vectors/md_flapping.svg';
+import { ReactComponent as md_paused } from './vectors/md_paused.svg';
+import { ReactComponent as md_resumed } from './vectors/md_resumed.svg';
+import { ReactComponent as md_unknown } from './vectors/md_unknown.svg';
+import { ReactComponent as md } from './vectors/md.svg';
+import { ReactComponent as placeholder } from './vectors/placeholder.svg';
+import { ReactComponent as servergroup_aws } from './vectors/servergroup_aws.svg';
+import { ReactComponent as sp_ci_branch } from './vectors/sp_ci_branch.svg';
```

Comment:
If you want to use svg icons for the CI views, they'll be available here. Feel free to continue to use the icon font though.

Reply:
could we add a line break + big comment block before all the legacy `sp` ones explaining what they are / the intended usage?

Might even be worth doing the same thing for different categories of icons (i.e. but all the MD ones in their own block, etc.)

Reply:
Great idea